sycl/include/sycl/detail/pi.h

```diff
@@ -74,9 +74,11 @@
 // PI_EXT_ONEAPI_CONTEXT_INFO_USM_MEMSET2D_SUPPORT, and
 // PI_EXT_ONEAPI_CONTEXT_INFO_USM_MEMCPY2D_SUPPORT context info query
 // descriptors.
+// 12.22 Added new bool argument to piextQueueCreateWithNativeHandle to select
+// between standard and immediate commandlists.
```

Comment:
this is ABI breaking change

Reply:
Yes it is. What else needs to be done to handle an ABI-breaking change?

Reply:
I think they are not allowed until 2024.0 
So you must retain the existing version and add new one

Reply:
Is there a precedent for making an ABI change similar to this one?
Should I add a new function named make_queue_x or something like that? Or is there a different way to handle it?